source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -18,3 +18,31 @@ The global vulnerability database is created automatically, currently pulling da
 This database can be configured to be updated periodically, ensuring that the solution will check for the very latest CVEs.
 
 Once the global vulnerability database (with the CVEs) is created, the detection process looks for vulnerable packages in the inventory databases (unique per agent). Alerts are generated when a CVE (Common Vulnerabilities and Exposures) affects a package that is known to be installed in one of the monitored servers. A package is labeled as vulnerable when its version is contained within the affected range of a CVE.
+
+Scan types
+==========
+
+The results are presented as alerts and also stored in a database. You can check the last scan alerts, or query each agent's individual CVEs DB.
+
+The Vulnerability Detector module can run a scan on startup (**run_on_start** option) and with a fixed frequency (**interval**).
```

Comment:
Can we set `run_on_start` and `interval` as linked references to the settings?

Reply:
All references to configuration parameters now are links too.